doc/method/method-intro.md

```diff
@@ -94,44 +94,27 @@ read out the reference taxonomy.
 As a simple example, consider a genus represented in both
 taxonomies, but containing different species in the two:
 
-> [this would be a figure]   S = (b,c,d)a,  S' = (c,d,e)a
+> S = (b,c,d)a,  S' = (c,d,e)a
 
 S and S' each have four nodes.  Suppose c, d, and a in S' are aligned
 to c, d, and a in S.  The only unaligned node is e, which is a
 sibling of c and d and therefore grafted as a child of a.  After the merge
 step, we have:
 
-> [figure] S + S' = (b,c,d,e)a
+> S + S' = (b,c,d,e)a
 
 One might call this merge heuristic 'my sibling's sibling is my
 sibling' or 'transitivity of siblinghood'.
 
-This is a very common pattern.  For example: in NCBI taxonomy, take a
-= _Bufo_, b = _B. spinosis_, c = _B. bufo_, d = _B. japonicus_, and
-in GBIF take a = _Bufo_, c = _B. bufo_, d = _B. japonicus_, e =
-_B. luchunnicus_.  There is no _B. spinosis_ in GBIF and no
-_B. luchunnicus_ in NCBI.  The _Bufo_ in the combined taxonomy has as
-its children copies of species records from both sources.  There are
+This is a very common pattern.  Figure 2a illustrates a real life-example when combining the genus _Bufo_ across NCBI and GBIF. There are
 hundreds of thousands of similar simple grafting events.
```

Comment:
I suppose I should verify this. There are 276375 nodes in OTT, but I don't know how many of them were built up with grafting. (The statistics don't separate grafts to grafts from grafts to preexisting workspace nodes, so can't calculate an exact number from info we already have.)  - not hard to count number of OTT nodes with children from multiple sources; counting graft events would require rerunning assembly with new instrumentation. So I would count OTT nodes and rewrite the sentence.